plugins/volumeFilter_test.go

```diff
@@ -1127,6 +1160,16 @@ func runTestVolumeFilterFn(
 func makeSellOpAmtPrice(amount float64, price float64) *txnbuild.ManageSellOffer {
 	return &txnbuild.ManageSellOffer{
 		Buying:  txnbuild.NativeAsset{},
+		Selling: testQuoteAsset,
+		Amount:  fmt.Sprintf("%.7f", amount),
+		Price:   fmt.Sprintf("%.7f", price),
+	}
+}
+
+func makeBuyOpAmtPrice(amount float64, price float64) *txnbuild.ManageSellOffer {
+	// Since a buy op buys the base and sells the quote, we reverse the assets and create a sell op.
+	return &txnbuild.ManageSellOffer{
+		Buying:  testQuoteAsset,
```

Comment:
we should be buying the base asset here and selling the quote -- base asset should be native.
the comment you've added here is helpful but we can remove the part after the comma.

context: the fact that it's a ManageSellOffer doesn't matter because we explicitly specify the buying and selling assets. It makes more sense when thinking about it in terms of base and quote.

the way kelp works (and feel free to add this line as a comment in makeBuyOpAmtPrice function if you find this context to be more helpful than confusing) is that all actions are performed in the context of the base asset. So a sell op will SELL the BASE asset. A buy up will BUY the BASE asset. It helps to name the assets as base and quote so it's easier to think in terms of base vs. quote instead of specific assets.

on the top can you add a variable called testBaseAsset setting that to the native asset to simplify things?

Reply:
This makes sense! Added a comment with some of this info, and put a `testBaseAsset` variable at the top.